src/AutoRest.CSharp/Mgmt/Generation/ResourceGroupExtensionsWriter.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using AutoRest.CSharp.Output.Models;
+using Azure.Core.Pipeline;
+
+namespace AutoRest.CSharp.Generation.Writers
+{
+    internal class ResourceGroupExtensionsWriter
```

Comment:
Can we add a test project that will generate `ResourceGroupExtensions` class and add tests?

Reply:
Added